yt_dlp/extractor/adobepass.py

```diff
@@ -1635,6 +1640,56 @@ def extract_redirect_url(html, url=None, fatal=False):
                         urlh.geturl(), video_id, 'Sending final bookend',
                         query=hidden_data)
 
+                    post_form(mvpd_confirm_page_res, 'Confirming Login')
+                elif mso_id == 'Suddenlink':
+                    # Suddenlink is similar to SlingTV in using a tab history count and a meta refresh,
+                    # but they also do a dynmaic redirect using javascript that has to be followed as well
+                    provider_redirect_page, urlh = provider_redirect_page_res
+
```

Comment:
unused?
```suggestion
```

Reply:
After reviewing this myself, `provider_redirect_page` var is unused. I was mostly keying off other examples in the code. Can be removed.